Mathlib/NumberTheory/Transcendental/AnalyticPart.lean

```diff
@@ -0,0 +1,606 @@
+/-
+Copyright (c) 2025 Michail Karatarakis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michail Karatarakis
+-/
+module
+
+public import Mathlib.Analysis.Analytic.Order
+public import Mathlib.Analysis.Complex.CauchyIntegral
+public import Mathlib.Analysis.Normed.Module.Connected
+
+/-!
+Some auxiliary lemmata covering the analytic part of the proof of the Gelfond–Schneider theorem.
+-/
+
+set_option autoImplicit true
+set_option linter.style.multiGoal false
+set_option linter.unusedVariables false
```

Comment:
Please don't disable linters like this (for the final draft of the code, at least).